lib/govuk_app_config/govuk_json_logging.rb

```diff
@@ -26,6 +26,7 @@ def self.configure
         "#{hash.to_json}\n"
       },
     )
+    Rails.logger = ActiveSupport::TaggedLogging.new(logger)
```

Comment:
Yeah totally agree, we ended up using `ActiveSupport::TaggedLoggging` because `ActiveSupport::Logger.new()` didn't immediately work as expected. I had another playaround with it and realized that the formatter needs to be set separately instead of part of the keyword arguments in order for it to work. The new version uses `ActiveSupport::Logger` instead of TaggedLogging.